packages/server/src/api/bannerRouter.ts

```diff
@@ -101,6 +105,15 @@ export const buildBannerRouter = (
                 variant.tickerSettings &&
                 tickerData.addTickerDataToSettings(variant.tickerSettings);
 
+            if (testTracking.campaignCode === 'PD-TEST_CONTROL') {
```

Comment:
Not sure if this should be `testTracking.campaignCode` or `test.name`, are they different values?

Reply:
I assume if there are multiple variants in the test this will change the heading for every variant? We should confirm we're not planning on A/B testing against the standard contributions banner, if we are then we'll want to check the variant name in this conditional to.

Reply:
Think PD-TEST is the test name, CONTROL the variant name hence reads 'PD-TEST_CONTROL' as a campaignCode. Your right, would be clearer to test these 2 separately and place what we are looking for in a easily changed constant for when in comes through from DRR (alongside country's header/highlight copy)